apps/backoffice-v2/src/pages/Entity/components/Case/case-action-variants/WebsiteMonitoringCaseActions/constants.ts

```diff
@@ -0,0 +1,9 @@
+import { StateTag } from '@ballerine/common';
+
+export const websiteMonitoringTagsToData = {
+  [StateTag.APPROVED]: { variant: 'success', text: 'Approved' },
+  [StateTag.REJECTED]: { variant: 'destructive', text: 'Rejected' },
+  [StateTag.MANUAL_REVIEW]: { variant: 'info', text: 'Manual Review' },
+  [StateTag.PENDING_PROCESS]: { variant: 'warning', text: 'Processing' },
+  [StateTag.FAILURE]: { variant: 'destructive', text: 'Failed' },
+} as const;
```

Comment:
Let's sync on this.

Reply:
P0

Reply:
This will be non relevant once this PR will be added to tabs feature and so will be deleted.